### PR TITLE
[#1151] Fix for extract metadata option

### PR DESCRIPTION
**Pull request by mauriel133**
block modal_add_file was duplicated, causing the error.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@zhiyuli Please review.

File: theme/templates/resource-landing-page/modals.html [hunk 1]
```diff
@@ -777,40 +777,6 @@ <h4 class="modal-title" id="extra_meta_title"> Add/Edit Extended Metadata </h4>
...
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                    <button type="submit" class="btn btn-primary">Save changes</button>
-                </div>
-            </form>
-        </div>
-    </div>
-</div>
-{%  endblock %}
-
```
Comment (hyi) [inline theme/templates/resource-landing-page/modals.html]: @mauriel133 If you remove this modal_add_file block from resource-landing-page/modals.html which is included in genericresource.html, will add files button still work for resource landing page? Will removing duplicate add file dialog from timeseriesresource.html be sufficient?

Comment (mauriel133) [inline theme/templates/resource-landing-page/modals.html]: It still works as I tested. The timeseriesresource.html rewrites this block in genericresource.html, so we don't want to delete that one.

Comment (hyi) [inline theme/templates/resource-landing-page/modals.html]: @mauriel133 OK, now I see this add file block is included in genericresource.html as well, so the duplicate one in this modals.html can be removed safely. a +1 from me.

**Pull request merged**